api/admin.go

```diff
@@ -387,7 +414,25 @@ func addCertificate(c *Context, w http.ResponseWriter, r *http.Request) {
 func removeCertificate(c *Context, w http.ResponseWriter, r *http.Request) {
 	props := model.MapFromJson(r.Body)
 
-	if err := app.RemoveSamlFile(props["filename"]); err != nil {
+	certificateFile := ""
+
+	switch certType, _ := strconv.Atoi(props["type"]); certType {
```

Comment:
Same problem here, this is a breaking change. Is there a way we can do this without breaking the endpoint?